src/resolvers-cassandra/Settings/mutations.js

```diff
@@ -86,6 +89,9 @@ function editSite(args, res) { // eslint-disable-line no-unused-vars
         ]
       }]);
     })
+    .then(() => {
+      serviceBusClient.sendMessages(SERVICE_BUS_CONFIG_QUEUE, JSON.stringify({'dirty': 'settings'}));
```

Comment:
This is really just sending a single message so maybe it should be renamed to `sendMessage`?

Reply:
I agree, this is part of `azure-sb` though, not my `ServiceBusClient`.